src/svg/strategies/SvgStrategyLinechart.ts

```diff
@@ -42,10 +42,18 @@ class SvgStrategyLinechart extends SvgStrategy {
         super();
         this.axes = new XYAxes();
         this.lines = new Lineset(this.axes.x, this.axes.y);
-        this.annotations = new Annotations(this.axes.x, this.axes.y);
+        // this.annotations = new Annotations(this.axes.x, this.axes.y);
         this.alerts = new Alerts(this.axes.x, this.axes.y);
     }
 
+    public addComponent(component: Function, config: any) {
```

Comment:
Why do you implement this method more than one time? I think the `addComponent()` method shouldn't be an abstract method. The implementation could be in the SvgStrategy class, instead of each child class.